### PR TITLE
PPTP-1209 Updated Get Subscription Status (API 1534) to v2.2.0

### PR DESCRIPTION
**Pull request by paulblakehmrc**
### Description of Work

Added channel to subscription status response

#### Check list 
 - [x] `./precheck` was executed (Integration/Component/Unit tests)
 - [x] `sbt scalafmt test:scalafmt` was executed
 - [ ] Required Environment Config has been amended/added
 - [ ] Links to dependencies have been included (BE/FE work)
 - [x] User Acceptance Tests (UAT) were run locally and have passed.
 - [ ] No Accessibility errors/warnings reported by Wave

### CODE REVIEW

File: test/spec/PptTestData.scala [hunk 1]
```diff
@@ -192,7 +193,8 @@ trait PptTestData extends RegistrationBuilder with MockAuthAction {
   protected val subscriptionStatus: SubscriptionStatus = SubscriptionStatus(
     subscriptionStatus = ETMPSubscriptionStatus.NO_FORM_BUNDLE_FOUND,
     idValue = "XXPPTP123456789",
-    idType = "ZPPT"
+    idType = "ZPPT",
```
Comment (SteveSugden) [inline test/spec/PptTestData.scala:196]: Worth removing the "Z" for consistency

**Review by SteveSugden — APPROVED** [review]
LGTM

File: test/spec/PptTestData.scala (hunk 1 above)
Comment (paulblakehmrc) [reply test/spec/PptTestData.scala:196]: The idType in the repsonse, here, should stay as `ZPPT`, as per the spec. It's the regime that's corrected from `ZPPT` to `PPT`.

Comment (SteveSugden) [reply test/spec/PptTestData.scala:196]: I assumed the "Z" was a typo!  :-)

**Pull request merged**